orbotservice/src/main/java/org/torproject/android/service/OrbotService.java

```diff
@@ -326,12 +329,23 @@ private void stopTorAsync() {
 
     }
 
-    private static boolean useIPtProxy ()
+    private static boolean useIPtObfs4Proxy ()
```

Comment:
This should be renamed since it includes meek as well

Reply:
Agreed